pkg/util/utils.go

```diff
@@ -10,7 +10,7 @@ import (
 	"strings"
 
 	mapset "github.com/deckarep/golang-set"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why alias here?

Reply:
My IDE automatically added it, I'm also confusing.. I'll manually delete it before next push